clusterloader2/config/codec.go

```diff
@@ -41,3 +46,16 @@ func ReadConfig(path string) (*api.Config, error) {
 	}
 	return &config, nil
 }
+
+// ReadTemplate creates object template from file specified by the given path.
+func ReadTemplate(path string) (runtime.Object, error) {
+	raw, err := ioutil.ReadFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("reading template error: %v", err)
```

Comment:
-> "error reading template file: "

Reply:
Should this type of error be returned  by ReadTemplate method or by ReadConfig method caller (as a wrapper for the error returned by ReadConfig)?

Reply:
Based on the scope of this error I would believe the former. In the outer function, the scope should be wider imo (for e.g sth like "couldn't fetch template: ")

Reply:
done